transfers/util.py

```diff
@@ -207,15 +207,18 @@ def make_location(row: pd.Series) -> Location:
     # created_at = row.DateCreated
 
     location = Location(
-        # nma_pk_location=row.LocationId,
+        nma_pk_location=row.LocationId,
+        # TODO: determine if PointID should map to location.name or thing.name or if the Location table needs a name field at all.
         name=row.PointID,
         point=transformed_point.wkt,
         release_status="public" if row.PublicRelease else "private",
-        # elevation_accuracy=row.AltitudeAccuracy,
-        # elevation_method=row.AltitudeMethod,
+        elevation_accuracy=row.AltitudeAccuracy,
+        elevation_method=row.AltitudeMethod,
         # created_at=created_at,
-        # point_accuracy=row.CoordinateAccuracy,
-        # point_method=row.CoordinateMethod,
+        coordinate_accuracy=row.CoordinateAccuracy,
+        coordinate_method=row.CoordinateMethod,
+        nma_coordinate_notes=row.CoordinateNotes,
+        nma_notes_location=row.LocationNotes,
```

Comment:
I can update this when I merge your work with the lexicon updates, but now that we are not using lookup table codes I think we'll need to amend this section (and for other tables/fields that are lexicon/lookup tables) to get the `MEANING` from the LU tables. Otherwise they won't be found in the lexicon term table and we'll run into errors.

On that note, I'll wait to open a PR for lexicon so that I can take care of that in the transfer scripts where it is relevant.